CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 2.6.8
+
+* Fix missing screenshot on failure when REST helper is in use #2513 by @PeterNgTr
```

Comment:
Please, update changelog by last master commits

Reply:
I will do that